drift/info_parser.py

```diff
@@ -1,36 +1,43 @@
-DRIFT_FACTS = {'bios_uuid', 'display_name'}
-DRIFT_SYSTEM_ID = {'id'}
+SYSTEM_ID = 'id'
 
 
 def build_comparisons(inventory_service_systems):
     """
     given a list of system dicts, return a dict of comparisons, along with a dict of metadata.
     """
-    fact_comparison = _select_applicable_info(DRIFT_FACTS, inventory_service_systems)
+    fact_comparison = _select_applicable_info(inventory_service_systems)
 
     metadata = [_system_metadata(system) for system in inventory_service_systems]
     return {'facts': fact_comparison, 'metadata': metadata}
 
 
-def _select_applicable_info(info_keys, systems):
+def _select_applicable_info(systems):
     """
     Take a list of systems, and output a "pivoted" list of facts, where each fact
     key has a dict of systems and their values. This is useful when comparing
     facts across systems.
     """
     # create dicts of system + info
-    ids_and_info = [{key: system[key] for key in (info_keys | DRIFT_SYSTEM_ID & system.keys())}
+    ids_and_info = [{**{key: _system_facts(system)[key] for key in _system_facts(system)},
```

Comment:
What does `**` do in this context? This is quite the list comprehension to mentally unpack and understand.

Reply:
It "unzips" the dicts so they are not nested when a new one is created. For example:
```
In [1]: x = {'foo': 'bar'}

In [2]: y = {'baz': 'quux'}

In [3]: {**x, **y}
Out[3]: {'foo': 'bar', 'baz': 'quux'}
```

I could also do `{key: _system_facts(system)[key]}[SYSTEM_ID] = system['id']`

Basically I just need the system ID added in alongside the rest of the facts for the transformation later on. I think I can rewrite this cleaner, gimme a few